dbt_semantic_interfaces/test_utils.py

```diff
@@ -38,8 +38,8 @@ def as_datetime(date_string: str) -> datetime.datetime:
 
 
 def find_semantic_model_with(
-    model: PydanticSemanticManifest, function: Callable[[SemanticModel], bool]
-) -> Tuple[SemanticModel, int]:
+    model: PydanticSemanticManifest, function: Callable[[PydanticSemanticModel], bool]
+) -> Tuple[PydanticSemanticModel, int]:
```

Comment:
I think this is the right thing to do for now but I suspect we should eventually move most of our test shims to use the protocols. We can open an issue or something for this later.